### ISSUE TITLE
1.19.4 adaptation of holograms [Future]

### ISSUE THREAD

**Issue by jwoff78** [DecentSoftware-eu/DecentHolograms#120]
### Just making sure

- [X] I have read the wiki and made sure, this feature is not yet in the plugin.
- [X] I believe, that this feature is possible to implement.
- [X] I made sure, that this feature has not been requested yet.

### Problem

I am unable to achieve '...'
I think it would be useful for '...'

### Feature

The implications of 1.19.4 allows you to completely replace normal armor stand workings, the new features have clickable entities, ridable entities, and more, a full description can be found here [Java_Edition_1.19.4_Non-mob_entities](https://minecraft.fandom.com/wiki/Java_Edition_1.19.4#Non-mob_entities) I've noticed that from 1.19.4 onward can completely change how people use holograms, using a block, text, item display would be a lot easy then using armor stand manipulation (for the onwards) they also have features such as rotation and scale which are not the most easy to produce as is

### Alternatives

It would also be possible by doing '...'

**Comment by Andre601** [comment]
We're well aware about the Display Entities that will be introduced in the upcoming MC release.

This will bring a lot of cool features, but at the same time a lot of work, especially when d0by wants to keep backwards compatability with the older MC releases, which obviously do not have these features.

As an example, the entities as of right now can...
- ...have their text rotation locked
- ...have the bg disabled
- ...have custom bg color
- ...have custom text size
- ...have text alignment

Those are only a few of the features and they all wouldn't be available in older versions (Or at least not without heavy work), so a separate setup would be needed to be made here to achieve support while keeping compat.

Other, smaller issues and inconveniences are things like how to support existing features of DH. For example, how should per-line permissions to hide/show them be supported with the new entity, if people also want to use text alignment? It would either require splitting up the lines into individual entities (Which is a waste of its line-break support) or add some logic to split the lines that do have special things like line height (Which would also cause issues), permission, etc.

tl;dr: We know about that feature, we want to support it, but it will take a lot of work and time to support it, so patience is apreciated.

**Comment by jwoff78** [comment]
I'm aware of backward compatibility, or just version compatibility issues in general, could be a sub feature set I am unsure, but glad to know =)

**Comment by SzczurekYT** [comment]
One way to go would be providing two types of holograms, the classic armor stand one and the new 1.19.4 one.
I could be even a new command, or as a subcommand.
This way player can use the new features, while keeping compatibility with older versions.

**Comment by Cyanoure** [comment]
I'm looking forward to this update

**Comment by d0by1** [comment]
Added in 2.10.0, see https://github.com/DecentSoftware-eu/DecentHolograms/releases/tag/2.10.0.
